api/src/main/java/org/apache/iceberg/Table.java

```diff
@@ -286,4 +286,18 @@ default AppendFiles newFastAppend() {
    * Returns a {@link LocationProvider} to provide locations for new data files.
    */
   LocationProvider locationProvider();
+
+  /**
+   * Returns the current refs for the table
+   *
+   * @return the current refs for the table
+   */
+  Map<String, SnapshotRef> refs();
+
+  /**
+   * Returns the snapshot ref with the given name or null if it does not exist
+   *
+   * @return the ref with the given name or null if it does not exist.
+   */
+  SnapshotRef ref(String ref);
```

Comment:
do we really need this method, if it's always `refs().get(ref)`

Reply:
oh nvm we also have the method in `TableMetadata`, so probably better to have it here

Reply:
Right, my thinking was just to keep it the same

Reply:
Yeah, I think it's nice to provide the convenience method.